src/lib.rs

```diff
@@ -470,9 +549,26 @@ impl TxBuilder {
         for (address, amount) in &self.recipients {
             tx_builder.add_recipient(to_script_pubkey(address)?, *amount);
         }
+        tx_builder.change_policy(self.change_policy);
+        if !self.utxos.is_empty() {
+            let bdk_utxos: Vec<BdkOutPoint> = self.utxos.iter().map(BdkOutPoint::from).collect();
+            let utxos: &[BdkOutPoint] = bdk_utxos[..].try_into().unwrap();
```

Comment:
I believe this is a cleaner way to convert your Vec to a slice.
```suggestion
            let utxos: &[BdkOutPoint] = &bdk_utxos;
```